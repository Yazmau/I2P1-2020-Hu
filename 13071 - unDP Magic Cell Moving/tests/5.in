1000
1 1 9935618
1 3 186519822
1 3 875475196
1 1 519691600
1 3 451325395
2 1
1 1 367238033
2 1
3 815690923
3 169763905
4
1 1 798187057
1 3 464917315
3 826778709
3 912376693
3 495483069
1 1 735210369
2 1
2 3
2 3
1 3 449439700
1 1 329055650
2 1
2 3
4
1 3 13713488
4
1 3 118386334
3 6760521
2 3
2 3
3 457613750
1 1 336289383
3 823108963
2 1
3 18318411
2 3
4
4
1 3 240861127
1 3 169004848
1 3 439037477
4
2 1
4
4
4
3 648402170
1 3 873292674
4
2 1
2 1
2 3
4
2 1
2 3
2 3
3 955486561
3 496106786
1 3 561290797
2 3
2 3
1 3 815511861
3 35534718
3 353251940
2 1
3 943252549
4
3 799532392
2 1
3 937813154
2 1
2 3
1 1 682139110
2 3
1 3 990715366
1 1 925896649
3 731338034
4
4
1 3 576557813
4
3 904918727
3 686856963
1 1 948566875
3 186161395
3 768935741
2 1
3 767477176
3 897498046
2 1
3 988993566
3 823431462
1 3 864002856
1 3 269487991
2 3
4
1 1 685198702
4
2 3
2 3
1 1 854473457
2 1
2 1
3 918522227
3 456891950
3 511191551
2 1
1 1 861235205
2 1
3 696760451
4
3 864293402
3 840774183
4
1 3 177071603
4
2 1
4
1 3 749258625
1 3 981599683
4
2 3
2 1
1 1 502715270
2 1
3 862972388
3 235275996
2 3
1 1 459012855
2 3
2 3
4
1 3 352943
2 1
4
3 560209645
1 1 624734169
2 1
1 1 216513444
2 3
3 284105638
4
1 1 238425917
1 1 673724317
3 189234534
2 1
1 3 205586845
3 655612552
2 1
2 3
3 13747564
2 3
2 3
2 3
1 3 528618443
3 534986329
1 1 716423892
4
4
3 660856019
3 463725293
1 1 724151622
1 1 540522780
3 503488952
4
2 1
2 3
2 1
1 1 772369557
4
2 3
4
2 1
3 677586552
4
4
2 1
2 3
3 27488268
1 3 7177012
2 1
2 1
2 1
2 3
4
1 3 227938366
2 1
2 3
1 1 541518439
1 1 338012476
3 968919055
2 1
1 3 858328987
2 3
1 3 986470229
1 3 366853204
4
3 253159084
3 786088920
3 329556255
3 472143442
2 1
1 3 879951028
2 3
3 485795661
2 1
2 3
3 899597756
4
3 572562725
4
2 1
1 1 433727414
3 854340945
2 1
1 1 690035755
1 1 375859713
1 1 524204854
4
2 3
2 1
1 1 797124112
1 1 277853547
1 1 443593462
3 396575133
1 3 82897580
1 1 806077889
4
1 1 232642428
2 3
1 3 876939519
4
1 1 196029077
4
1 1 772879838
4
1 3 999694182
4
1 3 143284304
2 1
4
1 3 616074785
3 456648509
4
1 3 612945475
3 269728103
3 843096053
2 3
1 3 864317430
1 3 206388636
2 3
1 3 82684405
2 3
2 1
4
1 3 92514529
2 1
2 3
3 859706926
2 3
4
1 3 618503155
1 3 418102089
2 3
1 3 484166978
1 1 144893293
4
2 1
3 740163179
3 276975731
2 3
1 1 554641693
4
3 252411813
4
3 146768752
4
2 1
2 3
2 3
3 114200490
4
2 1
1 3 904476584
3 57255513
3 70685275
4
4
4
1 1 489826305
1 3 185252081
3 484838652
2 1
1 3 597228851
2 3
3 312526379
4
2 3
2 3
4
4
2 1
3 658235292
3 474469972
3 57960304
2 1
3 25490019
4
4
3 705601259
3 951041391
4
2 1
4
4
2 3
3 386664297
4
3 88774348
2 1
3 170982237
3 316073620
4
1 1 670853837
3 902554461
1 3 991578248
1 1 873418660
3 436940983
4
4
2 1
1 1 945059164
1 3 132928626
4
1 1 985703275
3 523700075
3 157902165
3 581334046
4
3 879427599
4
4
2 1
4
2 1
1 3 296856159
2 3
3 230044054
1 1 768528706
4
1 1 17475219
1 1 985456452
2 1
1 1 334318212
1 3 964571152
4
4
4
2 3
4
2 1
4
3 789385235
3 830705990
2 3
3 116420900
1 3 126238183
3 743446694
3 361823922
2 3
1 3 631376691
1 3 389039234
1 1 970958424
2 1
1 1 620727436
1 1 687251358
3 832779608
1 1 494354176
3 138654608
4
3 463183694
2 3
3 255459188
4
4
2 1
4
1 3 137474925
4
4
1 1 380903430
3 376132955
2 3
3 347090752
2 1
4
1 1 949117793
3 160045268
1 1 989740429
4
3 839986866
2 1
4
4
3 168879578
3 534775296
1 3 245917995
3 574190122
1 3 874388475
2 3
2 3
2 1
1 3 969088098
1 3 292381593
4
2 1
4
4
3 656872483
1 3 545422412
4
1 1 534899369
1 1 955161759
2 1
4
3 539491718
1 3 626654042
2 3
4
4
2 3
4
4
2 1
3 801251525
4
1 1 912493118
3 864204307
3 785266548
1 1 107263248
4
1 1 245204112
3 744715782
4
2 3
3 168503463
3 246647315
4
4
1 3 466489315
3 735214309
2 1
3 857070198
1 3 905343570
3 451206741
4
3 137429394
4
4
4
4
4
2 1
2 3
1 3 121979621
3 87965050
4
1 3 832762856
2 1
3 822862615
4
2 3
2 1
2 3
4
4
4
1 3 166506858
4
3 749041181
1 1 8831172
3 499008499
4
4
3 781687720
4
2 3
1 1 69952331
1 3 750280582
2 1
1 3 250377813
2 3
3 407017852
4
3 829585983
1 3 18741598
3 882335053
1 1 568934072
1 1 594649626
2 1
3 385719410
2 3
1 1 103315720
2 1
2 1
3 552178186
4
3 811466670
4
3 745692016
1 3 54464533
3 941154185
2 3
1 3 197220041
3 897429299
1 3 825038235
1 1 943142775
1 3 684817296
2 1
1 3 288602538
3 648077593
2 3
1 3 280764173
2 1
3 99884408
3 246756193
1 1 996896291
4
3 175017843
1 1 657033221
2 3
2 1
4
2 1
2 3
4
1 3 714643000
3 352778819
3 676630111
4
2 1
3 859606700
3 942811078
1 3 769444485
4
2 3
2 1
3 914755441
3 611907775
2 3
2 3
2 1
1 3 561727113
2 3
4
4
4
4
2 1
4
2 1
3 27975559
1 1 636357962
3 62416168
1 1 754608468
2 3
3 549327477
1 1 81165329
2 1
1 3 664520951
4
3 842424350
3 442941433
2 3
3 454497005
4
1 3 368762028
3 155570026
3 790697813
3 241106510
1 3 680468875
4
4
3 760731647
1 1 230994614
3 833640144
4
3 332067279
1 3 290681605
3 369001545
4
2 3
3 523574290
1 1 249038612
3 319685140
3 311780904
3 435840452
2 3
3 98025402
3 681730432
2 3
1 1 294213111
3 431957369
1 3 710454596
3 854754127
2 3
2 1
3 935007093
4
2 1
1 1 426811315
1 3 265619536
2 3
4
1 3 625077516
1 3 273995230
4
1 3 175987834
1 1 246181519
1 1 41615624
1 3 823088402
3 857394676
4
3 64876992
1 1 669032176
4
1 3 957940426
4
1 3 272885225
1 1 618526657
1 3 238372921
4
1 3 314038680
1 1 571537066
4
3 202382374
4
3 632258850
4
2 1
4
1 1 115105368
3 499815534
4
3 938944994
4
4
1 1 622485180
4
4
3 114450444
4
1 3 798905502
2 1
2 3
1 1 497610062
2 1
3 798558978
3 728270372
3 127218549
1 3 714288794
2 3
3 660321488
4
1 3 532026370
4
2 3
2 1
4
1 3 395856331
4
4
1 3 969626482
3 402252844
4
4
3 239977425
2 3
3 877171717
2 3
2 3
3 959299523
3 669949370
2 1
1 3 163926797
2 3
2 1
1 1 413006208
3 151442433
1 1 684969320
1 1 270175605
2 3
3 681412563
3 824492944
2 1
2 1
3 16617196
2 3
3 547382291
1 3 695686377
2 3
2 1
4
2 1
1 3 813619071
1 3 527134446
3 966842894
1 3 664079831
4
3 288225425
2 1
2 1
4
2 1
3 741113569
1 3 50883014
1 3 824877257
4
4
4
2 3
2 3
1 1 958165380
2 1
3 943647149
2 1
3 937459377
4
2 3
1 1 157505550
1 3 268982660
2 1
1 1 403902245
1 1 695818804
1 1 325221253
1 3 333457574
3 30660433
4
1 1 129785650
2 3
2 1
3 835728148
2 3
2 1
3 239712850
2 1
4
4
1 1 1607627
1 1 481209520
4
4
4
1 3 819737372
2 1
3 287976353
4
4
4
4
4
3 861993163
3 745736896
1 1 252741583
2 3
1 1 767864607
4
1 3 877723520
1 1 185690666
3 689408883
1 3 307679766
4
3 91137007
1 1 448285706
3 792605639
2 3
2 1
2 1
4
4
3 759487413
1 3 357083564
3 47337680
4
3 626568767
4
3 802024536
1 3 610749798
1 1 536379669
1 3 301992733
2 1
1 3 306031901
3 216660445
1 1 548151008
2 3
3 177389610
4
4
3 336545095
1 1 717193902
4
2 3
4
3 932357053
1 3 642096014
1 1 295491838
2 1
2 3
1 3 363154723
1 3 581179888
3 215318473
3 507341332
3 463729452
3 290340956
2 1
4
1 3 510620848
1 1 565415958
2 1
4
4
1 3 171067083
2 3
1 3 165467907
4
1 3 470096015
4
3 726026225
1 3 768227923
1 1 804338678
1 1 433617998
3 737025400
3 883011097
3 968479343
2 1
4
1 3 525132200
2 3
4
4
1 1 525711817
4
1 1 520473101
1 3 244953531
4
3 648791164
4
1 3 984388904
4
2 1
3 525669708
2 1
3 528440423
3 272421510
4
4
4
4
2 3
1 1 919545716
3 564609111
3 562954932
4
4
3 643327500
1 3 469909044
1 3 746592425
3 43805354
3 532618378
2 1
3 600562222
3 616827491
3 485663790
1 1 431342396
4
4
4
1 3 722870738
2 1
4
3 62303398
2 3
3 744730488
2 3
3 856154926
1 3 578795023
2 3
2 1
2 3
1 3 347983508
1 3 273280984
4
4
1 1 621775166
2 1
4
3 597084474
1 1 159244387
4
3 444431558
4
4
3 568330328
3 239472323
3 455712110
4
2 3
4
2 1
3 996520169
1 1 237745348
3 886754887
4
4
1 3 945241469
1 3 20995744
2 3
2 1
2 3
4
1 3 459640533
3 769744140
2 1
4
4
1 3 628218192
1 1 520942421
4
4
3 588835319
3 178622605
1 3 66441982
3 123064313
3 363358193
3 412581137
3 896213136
1 1 357972144
3 852466742
4
2 1
4
3 159714702
4
3 596260461
3 699553876
2 3
4
4
2 1
2 3
1 1 671883720
4
2 1
2 3
2 3
1 3 75003492
4
3 804768811
4
3 342154825
4
4
4
2 3
3 322572800
1 3 324189984
4
2 3
2 1
2 3
2 3
2 1
4
3 830594973
1 1 66140775
3 267348907
4
2 1
2 3
2 1
4
1 1 22441313
3 103255478
1 3 649078723
3 845883362
4
3 991511630
3 598786193
4
3 806444806
1 1 642030625
1 3 789128515
4
4
4
4
1 3 249412308
3 228758309
3 917358005
