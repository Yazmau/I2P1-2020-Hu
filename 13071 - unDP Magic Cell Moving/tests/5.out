Invalid
Invalid
Invalid
169763905
Invalid
Invalid
Invalid
Invalid
9935618
Invalid
9935618
Invalid
Invalid
823108963
823108963
Invalid
Invalid
823108963
18318411
18318411
18318411
Invalid
648402170
6760521
Invalid
943252549
Invalid
Invalid
Invalid
731338034
731338034
Invalid
731338034
Invalid
Invalid
Invalid
897498046
Invalid
897498046
Invalid
Invalid
696760451
840774183
Invalid
840774183
798187057
Invalid
Invalid
798187057
Invalid
823431462
Invalid
840774183
Invalid
Invalid
284105638
Invalid
Invalid
Invalid
Invalid
Invalid
534986329
534986329
Invalid
Invalid
503488952
Invalid
823431462
503488952
677586552
677586552
Invalid
13747564
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
968919055
Invalid
899597756
572562725
Invalid
Invalid
433727414
Invalid
Invalid
Invalid
Invalid
Invalid
396575133
Invalid
Invalid
854340945
Invalid
854340945
Invalid
854340945
Invalid
854340945
Invalid
396575133
Invalid
456648509
Invalid
Invalid
Invalid
Invalid
572562725
Invalid
189234534
Invalid
Invalid
Invalid
Invalid
27488268
Invalid
27488268
252411813
146768752
114200490
Invalid
70685275
70685275
70685275
Invalid
Invalid
Invalid
312526379
768935741
768935741
25490019
25490019
951041391
276975731
276975731
386664297
316073620
Invalid
Invalid
Invalid
436940983
436940983
Invalid
Invalid
859706926
Invalid
581334046
879427599
879427599
854340945
Invalid
Invalid
230044054
Invalid
Invalid
Invalid
Invalid
843096053
843096053
843096053
230044054
843096053
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
138654608
255459188
255459188
463183694
Invalid
463183694
463183694
Invalid
376132955
Invalid
Invalid
160045268
830705990
830705990
Invalid
Invalid
Invalid
Invalid
839986866
574190122
574190122
Invalid
656872483
Invalid
Invalid
690035755
Invalid
656872483
656872483
839986866
839986866
801251525
Invalid
Invalid
785266548
Invalid
744715782
246647315
246647315
Invalid
Invalid
451206741
137429394
137429394
137429394
137429394
137429394
Invalid
87965050
Invalid
822862615
87965050
87965050
87965050
Invalid
87965050
Invalid
499008499
499008499
781687720
Invalid
Invalid
Invalid
407017852
Invalid
Invalid
Invalid
Invalid
552178186
811466670
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
246756193
Invalid
175017843
175017843
Invalid
676630111
Invalid
942811078
Invalid
897429299
897429299
897429299
897429299
676630111
Invalid
Invalid
Invalid
62416168
454497005
Invalid
Invalid
241106510
241106510
Invalid
833640144
Invalid
369001545
Invalid
Invalid
Invalid
935007093
Invalid
Invalid
935007093
Invalid
Invalid
935007093
Invalid
Invalid
Invalid
Invalid
857394676
Invalid
64876992
Invalid
64876992
Invalid
Invalid
Invalid
64876992
Invalid
Invalid
64876992
202382374
632258850
681730432
Invalid
499815534
938944994
938944994
Invalid
938944994
938944994
114450444
Invalid
Invalid
Invalid
660321488
Invalid
660321488
660321488
Invalid
660321488
660321488
Invalid
402252844
402252844
Invalid
Invalid
Invalid
Invalid
Invalid
547382291
Invalid
Invalid
Invalid
966842894
127218549
Invalid
Invalid
741113569
741113569
741113569
Invalid
937459377
Invalid
Invalid
Invalid
Invalid
Invalid
Invalid
30660433
Invalid
442941433
442941433
Invalid
Invalid
442941433
442941433
442941433
Invalid
287976353
287976353
287976353
287976353
287976353
Invalid
442941433
Invalid
Invalid
Invalid
689408883
Invalid
745736896
745736896
Invalid
47337680
626568767
Invalid
Invalid
Invalid
Invalid
177389610
177389610
Invalid
336545095
792605639
Invalid
Invalid
Invalid
Invalid
336545095
Invalid
Invalid
216660445
216660445
Invalid
Invalid
336545095
Invalid
336545095
Invalid
Invalid
Invalid
216660445
Invalid
968479343
968479343
Invalid
968479343
Invalid
Invalid
968479343
648791164
Invalid
648791164
272421510
272421510
272421510
272421510
Invalid
562954932
562954932
Invalid
Invalid
485663790
485663790
485663790
Invalid
431342396
Invalid
Invalid
Invalid
648791164
648791164
Invalid
856154926
Invalid
597084474
444431558
444431558
455712110
648791164
Invalid
886754887
886754887
Invalid
Invalid
648791164
Invalid
886754887
886754887
Invalid
Invalid
886754887
886754887
Invalid
Invalid
852466742
744730488
159714702
852466742
852466742
Invalid
852466742
Invalid
769744140
804768811
342154825
342154825
342154825
Invalid
322572800
239712850
Invalid
267348907
322572800
Invalid
Invalid
845883362
598786193
Invalid
Invalid
806444806
806444806
806444806
806444806
Invalid
